lambda/write-activity-log/write-activity-log.ts

```diff
@@ -0,0 +1,80 @@
+import { SQSEvent } from "aws-lambda";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import {
+  DynamoDBDocumentClient,
+  PutCommand,
+  PutCommandOutput,
+} from "@aws-sdk/lib-dynamodb";
+import {
+  SendMessageCommand,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { Activity, ActivityLogEntry } from "./models";
+
+const dynamoClient = new DynamoDBClient({});
+const dynamoDocClient = DynamoDBDocumentClient.from(
+  dynamoClient
+);
+
+const sqsClient = new SQSClient({});
+
+export const validateActivityLogEntry = (activityLogEntry: ActivityLogEntry): void => {
+  if (!(
+          activityLogEntry.user_id  !== undefined &&
+          activityLogEntry.session_id !== undefined &&
+          activityLogEntry.timestamp !== undefined &&
+          activityLogEntry.truncated !== undefined
+      )) {
+      throw new Error(`Could not validate activity log entry ${JSON.stringify(activityLogEntry)}`);
+  }
+};
+
+export const validateActivities = (activities: Activity[]): void => {
```

Comment:
Should we call this from `[validateActivityLogEntry](https://github.com/alphagov/di-account-management-backend/pull/104/files#diff-ef509ba4be319ceefbeef69794ebb6e1b02b814eea64b00b7fc3150aa2d08391R22)`

Reply:
yes